src/utils/setup_process.js

```diff
@@ -0,0 +1,40 @@
+
+/*
+Copyright: Ambrosus Technologies GmbH
+Email: tech@ambrosus.com
+
+This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+This Source Code Form is “Incompatible With Secondary Licenses”, as defined by the Mozilla Public License, v. 2.0.
+*/
+import * as Sentry from '@sentry/node';
+import {WinstonConsoleLogger} from './loggers';
+import config from '../../config/config';
+
+const reportAndExit = (logger, message, error) => {
+  Sentry.captureException(error);
```

Comment:
I would assume this needs some time to send the error msg to Sentry. Is this synchronous? If not, the request will never go through because of the process.exit(1) just 2 lines below.

Reply:
It is not synchronous, but there is a way to flush the buffer. I saw an example in the docs on how to flush the queue and then exit on the completion callback.

Reply:
@conorseabrook do you have a link?

Reply:
Found it https://docs.sentry.io/error-reporting/configuration/draining/?platform=javascript thanks ;)

Reply:
👍 I was on my phone or I would have posted.